ansible/configs/sap-hana/post_software.yml

```diff
@@ -108,7 +108,7 @@
         - "user.info: ssh cloud-user@bastion-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}"
         - "user.info: "
         - "user.info: Make sure you use the following RSA key while connecting to the Bastion Host:"
-        - "user.info: {{ lookup('file', '{{ output_dir }}/{{ guid }}_infra_ssh_key.pem') | trim }}"
+        - "user.info: {{ lookup('file', '{{ output_dir }}/{{ guid }}_infra_ssh_key.pem').split('\n') }}"
```

Comment:
I don't believe this will be picked up by cloudforms as you might expect... Is this for Red Hat Summit? If so, then we should get you switched over to using `agnosticd_user_info`. I will reach out directly...

Reply:
Here is an example of how this may look using the new `agnosticd_user_info` module: https://github.com/redhat-cop/agnosticd/blob/sap-hana-user-info/ansible/configs/sap-hana/post_software.yml#L85-L123

Reply:
@jkupferer no, this is not for Summit. Can I still use agnosticd_user_info or do you recommend any other approach?

Reply:
I still recommend using `agnosticd_user_info`. It is compatible with the older `debug` module approach and eventually we are going to move everything over anyway. This will just put your config ahead of the curve :)

Reply:
Cool, will do that then